### PR TITLE
UHF-9190 Maternity and child healt clinic index

### PR DESCRIPTION
**Pull request by j-mys**
# [UHF-9190](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9190)

## What was done
Added an index for maternity and child health clinics

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout  UHF-9190-maternity_search`
  * `make fresh`
* Pull the related PR branch
* Run `make drush-cr`

## How to test

* [x] Run `drush-sapi` -commands `drush sapi-rt; drush sapi-c; drush sapi-i`
* [x] Check that index returns correctly https://elastic-helfi-sote.docker.so/maternity_and_child_health_clinic/_search?size=10&pretty=true

## Designers review
* [x] This PR does not need designers review

[UHF-9190]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9190?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Other PRs
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/666

### CODE REVIEW

**Comment by j-mys** [comment]
Seems like the address isn't quite right yet, have to revisit that

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "drupal/helfi_azure_fs": "^2.0",
         "drupal/helfi_drupal_tools": "dev-main",
         "drupal/helfi_navigation": "^2.0",
-        "drupal/helfi_platform_config": "^4.0",
+        "drupal/helfi_platform_config": "^4.2",
```
Comment (Jussiles) [inline composer.json:19]: Revert changes from this file. We don't update `helfi_*` modules or usually any other modules in Drupal instance's composer.json. Updates are done automatically by Github workflows when a new version of the module is released.

File: composer.lock
Comment (Jussiles) [inline composer.lock:1]: Remove this file from PR. We don't usually commit composer.lock changes in Drupal instance repo. Updates are done automatically by Github workflows when a new version of the module is released.

**Review by Jussiles — CHANGES_REQUESTED** [review]
Good work in overall! I added few comments here and in Slack that should be fixed before you can merge.

File: composer.json (hunk 1 above)
Comment (j-mys) [reply composer.json]: Removed

File: composer.lock
Comment (j-mys) [reply composer.lock:1]: Removed

**Pull request merged**